babel.config.json

```diff
@@ -1,11 +1,18 @@
 {
   "presets": [
     ["@babel/env", {
+      "loose": true,
```

Comment:
Original code:
```ts
ref.current?.focus({ preventScroll: true });
```
With `loose` enabled:
```js
(_ref$current = ref.current) == null ? void 0 : _ref$current.focus({
```
With `loose` disabled:
```js
(_ref$current = ref.current) === null || _ref$current === void 0 ? void 0 : _ref$current.focus({
```

In non-loose mode, the value is explicitely checked for both `null` and `undefined`, as it needs to work with `document.all`, but we don't use `document.all` anywhere so it's safe.
![image](https://user-images.githubusercontent.com/567105/98591158-bd9dbe00-22c7-11eb-8caf-9108ae9d51cb.png)

It might matter in the future for new js syntax features, but `loose` will only break edge cases so I think it's fine.